.github/workflows/pr.yaml

```
@@ -61,6 +61,7 @@ jobs:
       files_yaml: |
         test_cpp:
           - '**'
+          - '!.devcontainer/'
```

```suggestion
          - '!.devcontainer/**'
```

Same for all others too